daffodil-lib/src/main/scala/org/apache/daffodil/cookers/EntityReplacer.scala

```diff
@@ -281,7 +281,7 @@ final class EntityReplacer {
    * this form, but allow the others.
    */
   private def replaceEntity(proposedEntity: String, orig: String, context: Option[ThrowsSDE], forUnparse: Boolean,
-    allowByteEntity: Boolean): String = {
+                            allowByteEntity: Boolean): String = {
```

Comment:
Check your IDE settings for indentation. You've got a few places wher indentation was changed, but shouldn't be.